docs/_includes/content.md

```diff
@@ -1125,6 +1125,7 @@ In addition to manually setting environment variables, cargo-make will also auto
 * **CARGO_MAKE_PROFILE** - The current profile name in lower case (should not be manually modified by global/task env blocks)
 * **CARGO_MAKE_ADDITIONAL_PROFILES** - The additional profile names in lower case, seperated with a ';' character (should not be manually modified by global/task env blocks)
 * **CARGO_MAKE_CARGO_PROFILE** - The [cargo profile](https://doc.rust-lang.org/cargo/reference/manifest.html#the-profile-sections) name mapped from the **CARGO_MAKE_PROFILE** (unmapped value will default to CARGO_MAKE_PROFILE value)
+* **CARGO_MAKE_PASS_PROFILE** - Pass profile down to workspace member's spawned tasks.
```

Comment:
cargo make doesn't set this one. users do. so i do'nt think it should be in this list. but... i can take care of the docs later if you wish.
if you do want to write about it, try in the workspace support section maybe?

Reply:
I moved it. Please feel free to give any advice or improve it, I'm not quite sure of the quality.